app/Core/Ldap/User.php

```diff
@@ -105,6 +105,17 @@ protected function getGroups(Entry $entry, $username)
         return $groupIds;
     }
 
+    /**
+     * Return true to set new user with role as Manager
+     *
+     * @access protected
+     * @return boolean
+     */
+    protected function userDefaultRole()
```

Comment:
It's probably more explicit to call this function something like `getDefaultUserRole()` and move the new condition here.

```php
protected function getDefaultUserRole()
{
    if (LDAP_USER_DEFAULT_ROLE_MANAGER) {
        return Role::APP_MANAGER;
    }
    return Role::APP_USER;
}
```

Reply:
Hi, thank you for your quick reply. Yes do it!